### DISCUSSION TITLE
graphscope 下使用graphlearn 在大图上跑分布式训练

### DISCUSSION THREAD

**Discussion by LightCong** [alibaba/GraphScope#1263] [Q&A] [answered]
请教一个问题：使用vineyard 分布式的存一张大图，在大图之上跑分布式训练， 每个trainer 理论上讲都可以touch 到全图的数据，这样是否会导致 vineyard 数据分片在不同节点间频繁拷贝？

**Comment by welcome** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template! And a maintainer will get back to you shortly!
Please feel free to contact us on [DingTalk](https://h5.dingtalk.com/circle/healthCheckin.html?dtaction=os&corpId=dingb4b76e0fa9ccd8d516fd0f0acdf19f55&68e71190=0fe34ec0&cbdbhh=qwertyuiop), WeChat account(graphscope) or [Slack](http://slack.graphscope.io/). We are happy to answer your questions responsively.

**Comment by sighingnow** [comment, marked as answer]
> 

每个trainer是通过grpc连接到多个server上来touch到全局的数据的，每个server管理自己的partition的数据，fragment本身不会copy。
